src/components/scene/keyboard-shortcuts.js

```diff
@@ -0,0 +1,35 @@
+var registerComponent = require('../../core/component').registerComponent;
+var THREE = require('../../lib/three');
```

Comment:
Do we want to make this non configurable as well? It would be good for to keep these basic keyboard shortcuts consistent across all aframe experiences.